components/ide-service/pkg/server/ideconfig.go

```diff
@@ -51,6 +52,9 @@ func ParseConfig(ctx context.Context, b []byte) (*config.IDEConfig, error) {
 		return nil, err
 	}
 
+	configCatClient := experiments.NewClient()
+	experimentalIdesEnabled := configCatClient.GetBoolValue(ctx, "experimentalIdes", false, experiments.Attributes{})
```

Comment:
Should not it be per a user? 🤔 otherwise it either enables for all or none?

Reply:
It should, do you know where to get the user data info from?

Reply:
It should be done in getConfig then, i.e. provide user id and email similar to resolveWorkspaceConfig.